dashboard/db/migrate/20170502231246_create_courses.rb

```diff
@@ -0,0 +1,11 @@
+class CreateCourses < ActiveRecord::Migration[5.0]
+  def change
+    create_table :courses do |t|
+      t.string :name
+      t.integer :plc_course_id
```

Comment:
The Course table has a `plc_course_id` column which points to a `Plc::Course` object. Are you suggesting it should be called `course_id` instead?